tests/server/webhooks/test_auth.py

```
@@ -0,0 +1,151 @@
+@pytest.mark.asyncio
+async def test_omitted_scope_not_sent():
+    endpoint = _TokenEndpoint()
+    async with _client(endpoint) as client:
+        provider = OAuth2TokenProvider(_config(scope=None), client)
+        await provider.get_token()
+    assert "scope=" not in endpoint.last_request.content.decode()
+
+
+@pytest.mark.asyncio
+async def test_rate_limited_raises(monkeypatch):
+    async def _unexpected_sleep(_):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_unexpected_sleep`

Add return type annotation: `None`

(ANN202)